src/app/register/register.component.html

```diff
@@ -83,6 +83,15 @@
         </div>
       </form>
       <div class="actions">
+        @if (flags.friendlyCaptchaEnabled) {
+          <!-- Sitekey below is shared between environments and necessarily exposed in FE, so should be fine to hard-code -->
+          <div
+            class="frc-captcha"
+            data-sitekey="FCMIOJ2ARSHLBGAJ"
+            data-puzzle-endpoint="https://api.friendlycaptcha.com/api/v1/puzzle, https://eu-api.friendlycaptcha.eu/api/v1/puzzle"
+            #frccaptcha
```

Comment:
Is there an established use of `frc` we're trying to match? This name & CSS class seems slightly confusing if the 'c' also stands for 'captcha'. `#friendlyCaptcha` clearer?

Reply:
This is pasted from the [Full example in Angular](https://docs.friendlycaptcha.com/#/widget_api?id=full-example-in-angular) in the docs. Can change it but especially since the ticket is to prototype rather than definitely use I was prioritising doing things the quick way.